package/dropbear/0005-libdropbear.pc.in-added.patch.conditional

```diff
@@ -0,0 +1,14 @@
+--- a/libdropbear.pc.in	1970-01-01 05:30:00.000000000 +0530
```

Comment:
Patch needs appropriate format-patch header, including commit log providing a reason for the patch. For more info see: https://git-scm.com/docs/git-format-patch

Reply:
Yes, made the corrections and now created the patch with git format-patch header.